spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper'
+
+describe PasswordResetsController do
+  describe "POST create" do
+    context "with blank input" do
+      it "redirects to the forgot password page" do
+        post :create, email: ''
+        expect(response).to redirect_to forgot_password_path
+      end
+
+      it "sets the error message" do
+        post :create, email: ''
+        expect(flash[:danger]).to eq("Email field cannot be blank.")
```

Comment:
You don't want your spec tied to the implementation. Asserting `be_present` should be good enough.